user_data/notebooks/analysis_example.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
The notebook works great.
3 minor things:
* the 2nd code cell (3rd cell overall) uses tabs as intention, while the rest of the notebook uses spaces.
This may lead to odd results if users start copy/pasting code around.
* I would not default interval to "1m" ... this interval is pretty unreliable on low-volume pairs, and causes a lot of "outdated ticker" in dry/live modes. "5m" is used as default throughout the bot - so i would use that here too.
* the last cell in the notebook should be markdown, not code.

Reply:
fixed